src/main/views/includes/macros/form.njk

```diff
@@ -19,6 +19,27 @@
   {{ input(label, name, form, hint, 'text', inputClass, autocomplete) }}
 {% endmacro %}
 
+{% macro poundInput(label, name, form, hint = '') %}
+  {% set error = form.errorFor(name) %}
```

Comment:
Would be possible to use `input` here? I know there are some differences. Maybe we could make `input` macro more configurable to introduce things like:
- max length
- length
- wrapper class (for `<span class="input-symbol-pound">`.

Reply:
I think it can already do some of that, but having a specific macro for this is better IMO
Its reusable and it configured correctly

Reply:
I have talked to Tarik and "pound input" will be used in many places in the future.

I'm not sure if there is any gain of adding more arguments to input / any other macro and make implementation of it so much more complex. Maintaining of this wouldn't be easier.

To be honest I'd rather create a ticket for GOV template so that pound input would just have css class on input, not the whole structure of form implemented differently.